test/quantization/core/test_workflow_module.py

```diff
@@ -400,7 +400,7 @@ def test_zero_numel(self):
             x = obs(x)
 
     def _test_memoryless(self, obs_class):
-        obs = obs_class(memoryless=True)
+        obs = obs_class(dtype=torch.float32, compute_dtype=obs_class().dtype)
```

Comment:
if we are changing the user facing API, can we also rename the test case?

Reply:
it still creates a memoryless observer, its just hidden from the API